build-logic/project-plugins/src/main/kotlin/com/hedera/hashgraph/gradlebuild/lifecycle/LifecycleSupport.kt

```diff
@@ -0,0 +1,30 @@
+package com.hedera.hashgraph.gradlebuild.lifecycle
+
+import org.gradle.api.Project
+
+class LifecycleSupport {
+    companion object {
+        @JvmStatic
+        fun configureLifecycleTask(project: Project, taskName: String) {
```

Comment:
You could make this an extension method on `Project`:

```
fun Project.configureLifecycleTask(taskName: String) {
```

Then on the call sites you can just do this, because you all ready in the _Project_ scope: 

```
configureLifecycleTask("jacocoTestReport")
```

You could also remove the `@JvmStatic`. It's not needed for Gradle....

Reply:
Thats a good idea, let me open a follow up PR for this change.